test/ccd2rgb.jl

```diff
@@ -0,0 +1,31 @@
+@testset "ccd2rgb" begin
+    if !isfile("data/casa_0.5-1.5keV.fits")
```

Comment:
*Always* use `joinpath` when creating a path

Reply:
See https://github.com/JuliaAstro/Reproject.jl/commit/2fde8229a240e0818eb95e1b539342f13b2c7477#diff-75b23cc61d190dd99feab17b8e297ee3

Reply:
Also, please add the test data directory to Travis cache, like this: https://github.com/JuliaAstro/Reproject.jl/blob/a807b2f8034200f4fcd6266ac4cc32a698e09303/.travis.yml#L15-L16